src/constants/chainMetadata.ts

```diff
@@ -9,32 +9,40 @@ export const CHAIN_METADATA: {
     name: 'BNB mainnet',
     logoSrc: bnbLogo,
     explorerUrl: 'https://bscscan.com',
+    layerZeroScanUrl: 'https://layerzeroscan.com/',
     blockTimeMs: 3000,
     blocksPerDay: 28800,
     corePoolComptrollerContractAddress: '0xfD36E2c2a6789Db23113685031d7F16329158384',
+    nativeCurrencySymbol: 'BNB',
```

Comment:
To expend the potential use cases for this new field, I'd store the native token directly.
To do that, I'd recommend adding a new `getToken` utility function to the `tokens` package, which could look like this:

```typescript
import { ChainId } from 'types';

import { tokens } from '../../infos/commonTokens';

export interface GetTokenInput {
  chainId: ChainId;
  symbol: string;
}

export const getToken = ({ chainId, symbol }: GetTokenInput) => {
  const chainTokens = tokens[chainId];
  return chainTokens.find(token => token.symbol === symbol);
};
```

You could then use it in the chain metadata:
```suggestion
    nativeToken: getToken({ chainId: ChainId.BSC_MAINNET, symbol: 'BNB' }),
```

With this function built, you could also update the `useGetToken` hook so it uses it in order to reduce repetition.